### PR TITLE
docs(root): adding docs for versioning and releases

### PR DESCRIPTION
**Pull request by jd239**
## Summary of the changes
New page within the guidance site detailing the teams approach to releases and versioning.

## Related issue
#20 

## Checklist
- [x] I have manually accessibility tested any changes, if relevant.

### CODE REVIEW

File: src/content/structured/get-started/releases-versioning.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Releases will be triggered once the `Develop` branch pull request is merged into the `Main` branch. The package versions will follow the [semantic versioning](https://semver.org/) scheme.
+
+```js
+major.minor.patch
+```
+
+<br />
+
+**Major releases** will contain breaking changes to the component libraries. Breaking changes will be communicated to the community as early as possible to allow teams to plan when to integrate the new packages into their projects. Major release will be very infrequent with a least a year between major releases.
```
Comment (GCHQ-Developer-530) [inline src/content/structured/get-started/releases-versioning.mdx:23]:
```suggestion
Major releases will be very infrequent with at least a year between major releases.
```

Comment (GCHQ-Developer-530) [inline src/content/structured/get-started/releases-versioning.mdx:23]:
```suggestion
**Major releases** will contain breaking changes to the component libraries. Breaking changes will be communicated to the community as early as possible to allow teams to plan when to integrate the new packages into their projects. Major releases will be very infrequent with at least a year between major releases.
```

File: src/content/structured/get-started/releases-versioning.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+path: "/get-started/releases-versions"
+
+date: "2022-11-21"
+
+title: "Releases and versions"
+
+subTitle: "Our approach to releases and versioning."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/releases-versions.mdx"
```
Comment (ad9242) [inline src/content/structured/get-started/releases-versioning.mdx:10]: should be releases-versioning.mdx

**Review by ad9242 — CHANGES_REQUESTED** [review]

File: src/content/structured/get-started/releases-versioning.mdx (hunk 2 above)
Comment (ad9242) [inline src/content/structured/get-started/releases-versioning.mdx:10]:
```suggestion
contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/releases-versioning.mdx"
```

File: src/content/structured/get-started/releases-versioning.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+This page provides details on how the `@ukic/` packages will be released and versioned when published for Open Source.
```
Comment (GCHQ-Developer-112) [inline src/content/structured/get-started/releases-versioning.mdx:13]:
```suggestion
This page provides details on how the `@ukic/` packages will be released and versioned when published for open source.
```

File: src/content/structured/get-started/releases-versioning.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Releases will be triggered once the `Develop` branch pull request is merged into the `Main` branch. The package versions will follow the [semantic versioning](https://semver.org/) scheme.
```
Comment (GCHQ-Developer-112) [inline src/content/structured/get-started/releases-versioning.mdx:15]:
```suggestion
Releases will be triggered once the `develop` branch pull request is merged into the `Main` branch. The package versions will follow the [semantic versioning](https://semver.org/) scheme.
```

**Review by GCHQ-Developer-112 — APPROVED** [review]

**Review by ad9242 — APPROVED** [review]

**Pull request merged**